### PR TITLE
Maintenance/simplified documentation

### PR DESCRIPTION
**Pull request by EluciusFTW**
No new doc in this PR, just some cleanup and reduction of duplication.

### CODE REVIEW

File: src/spectrecoff-cli/commands/CanvasImage.fs [hunk 1]
```diff
@@ -33,38 +33,35 @@ type CanvasImageExample() =
...
     interface ICommandLimiter<CanvasImageSettings>
 
     override _.Execute(_context, _settings) =
-        Cli.Theme.setDocumentationStyle
-        BL |> toConsole
-        pumped "CanvasImage module"
-        |> alignedRule Left
-        |> toConsole
+        setDocumentationStyle
+        docHeader "CanvasImage module" |> toConsole
```
Comment (DanielmTheDev) [inline src/spectrecoff-cli/commands/CanvasImage.fs:44]: I think you might have overlooked this:

image

File: src/spectrecoff-cli/commands/BreakdownChart.fs [hunk 2]
```diff
@@ -33,36 +30,31 @@ type BreakdownChartExample() =
...
             BI [
                 P "breakdownChart: ChartItem list -> OutputPayload"
             ]
             BL
-            Many [C "The"; P "ChartItem"; C "union type consists of two options:"]
+            C "The"; P "ChartItem"; C "union type consists of two options:"
             BI [
-                Many [P "ChartItem:"; C "Consists of the label and a value for the item."]
-                Many [P "ChartItemWithColor:"; C "Additionally defines a color the item will be rendered in."]
+                define "ChartItem" "Consists of the label and a value for the item."
```
Comment (DanielmTheDev) [inline src/spectrecoff-cli/commands/BreakdownChart.fs:51]: I was first a little sceptical about those utility methods, thinking they might add more confusion than is worth small loss of duplication. 
But on second thought, I think they are quite readable and make for more expressive strings (as does the output module in fact :))

**Review by DanielmTheDev — APPROVED** [review]

File: src/spectrecoff-cli/commands/BreakdownChart.fs (hunk 2 above)
Comment (EluciusFTW) [reply src/spectrecoff-cli/commands/BreakdownChart.fs:51]: Well, thaey are only for the documentation, NOT for the examples above. But they do illustrate for users, how easily they can create composite functions for tempalting their output. Also, using this, we can actually easily change our documentation layout globally, by just tweaking functions, not having to revisit 20 files.

File: src/spectrecoff-cli/commands/CanvasImage.fs (hunk 1 above)
Comment (EluciusFTW) [reply src/spectrecoff-cli/commands/CanvasImage.fs]: Good catch, removed.

**Pull request merged**